mezzanine/pages/admin.py

```diff
@@ -105,24 +66,16 @@ def add_view(self, request, **kwargs):
         return super(PageAdmin, self).add_view(request, **kwargs)
 
     def change_view(self, request, object_id, **kwargs):
-        """
-        For the ``Page`` model, check ``page.get_content_model()``
-        for a subclass and redirect to its admin change view.
-        Also enforce custom change permissions for the page instance.
-        """
+        """ Enforce custom permissions for the page instance. """
```

Comment:
Nit: Can we keep docstring format consistent with the rest of the project? We always put the triple quotes on separate lines, even for one-line docstrings. Whether it's right is arguable, but it's consistent.